src/pages/saksbehandling/revurdering/OppsummeringPage/RevurderingOppsummeringPage.tsx

```diff
@@ -204,11 +205,17 @@ const RevurderingOppsummeringPage = (props: {
         }
     }, [props.revurdering.id]);
 
+    const HenterInnholdLocal = () => (
+        <div className={styles.henterInnholdContainer}>
+            <HenterInnhold />
```

Comment:
Hvis den komponenten tar inn classname, så kan vi bruke komponenten direkte på 217 i steden for å wrappe det sånn som her